docs/topic-guides/kotlin/suggest.md

```diff
@@ -0,0 +1,97 @@
+# Suggest
+
+The API for the `SuggestStore` can be found in the [MozillaComponents Kotlin documentation](https://mozilla.github.io/application-services/kotlin/kotlin-components-docs/mozilla.appservices.suggest/-suggest-store/index.html).
+
+> The `SuggestStore` is a synchronous, which needs to be wrapped in the asynchronous primitive of the target language you are using it in.
```

Comment:
I'd normally want this to be expanded, but I think it's okay for now since we plan to update our async handling in the near future.